cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -0,0 +1,43 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _application-logback:
+
+====================
+Application Logbacks
+====================
+
+YARN containers spawned by a CDAP application use the default container logback, ``logback-container.xml``,
+packaged with CDAP. This ``logback.xml`` does log rotation, and depending on the use case, may be sufficient.
+However, it is also possible to specify a custom ``logback.xml`` for a CDAP application.
```

Comment:
However, it is also possible to use a custom ``logback.xml`` for a CDAP application by packaging it with the application in ``src/main/resources`` directory.